.rspec

```diff
@@ -1,3 +1,4 @@
 --format documentation
 --color
 --tty
+--fail-fast
```

Comment:
@trevor-vaughan can you let us know why you are adding this flag? It seems like if you want faster feedback while iterating on spec tests, you should run only the tests you care about by passing in a file name. When this runs in CI, we will want to see all test errors, not just the first one encountered.

Reply:
Sorry about that. I'll pull that back out.